merlin.spec

```diff
@@ -126,7 +126,7 @@ Requires: libdbi1
 Requires: python-mysql
 %else
 %if 0%{?rhel} >= 8
-BuildRequires: python2-PyMySQL
```

Comment:
Are we not using `python2-PyMySQL` in the build process? No such code is removed here..
%check section seem to be running some python tests.
Mock (the build tool) will not install any `Requires` deps in the mock environment, only `BuildRequires`, so maybe we need both?

Reply:
I was confused by this also. My reasoning was that the requirement should be the same as for MySQL-python on EL7, do you disagree?

The safe bet would indeed be to do both, but then I'd argue we should do the same for the EL7 dependency.

It does seem to build green without it, however.

Reply:
I'm confused too, I thought it was required, since I added it a while back. But it doesn't seem so..

I agree that it should be the same as for el7. So we're good to go :)